### PR TITLE
Create _github-pages-challenge-daglaroglou.dag.json

### PR DESCRIPTION
**Pull request by daglaroglou**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/_github-pages-challenge-daglaroglou.dag.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "daglaroglou",
+        "email": "<christos.daglaroglou@gmail.com>"
```
Comment (Stef-00012) [inline domains/_github-pages-challenge-daglaroglou.dag.json:4]: Invalid socials, Remove the `<>` from the email field

File: domains/_github-pages-challenge-daglaroglou.dag.json
Comment (Stef-00012) [inline domains/_github-pages-challenge-daglaroglou.dag.json:1]: the `dag` subdomain does not exist so you can not create a subdomain of that

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by daglaroglou** [comment]
i do own it, i have it linked on discord as well but i dont have it pointing anywhere yet

**Comment by orxngc** [comment]
No, you don't own it.
image
Register the subdomain first before we can merge this.

**Comment by daglaroglou** [comment]
> No, you don't own it. image Register the subdomain first before we can merge this.

i have it linked in discord, i did the steps in the docs to link it there and i made a _discord........json file
image

**Comment by Stef-00012** [comment]
> > No, you don't own it. image Register the subdomain first before we can merge this.
> 
> i have it linked in discord, i did the steps in the docs to link it there and i made a _discord........json file image
> 
> can i make a dag.json file as well even if i have it for discord? if yes what should i include?

are you sure that it's verified? discord will still show the domain to yourself but if it isn't verified others won't see it (also you'll get a message in the connections)

**Comment by daglaroglou** [comment]
image
i see this in connections

**Comment by CuteDog5695** [comment]
You know that wasn't supposed to be merged, right?
If you don't want us to remove the domain which has the discord txt then please make dag.json (with your website's records)

**Comment by daglaroglou** [comment]
> You know that wasn't supposed to be merged, right? If you don't want us to remove the domain which has the discord txt then please make dag.json (with your website's records)

i dont have a website right now, i host it in github pages, so how can i link that there without the special guide of gh pages?

**Comment by orxngc** [comment]
Closing this PR as stale for now. If you want to know how to register your site with Github Pages, follow the [documentation](https://www.is-a.dev/docs/github-pages/).

**Pull request closed without merging**